python/tests/test_tables.py

```diff
@@ -1309,6 +1309,16 @@ def test_add_row_bad_data(self):
             t.add_row(metadata=[0])
 
 
+class TestTableCollectionIndex:
+    def test_index(self):
+        i = np.arange(20)
+        r = np.arange(20)[::-1]
+        index = tskit.TableCollectionIndex(edge_insertion_order=i, edge_removal_order=r)
```

Comment:
Should probably be ``TableCollectionIndexes`` for consistency

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/916/commits/150dad1251e1329ce97824161e96a7e18c5ac60b